src/test/kotlin/org/rust/ide/inspections/RustInspectionsTest.kt

```diff
@@ -23,6 +23,19 @@ class RustInspectionsTest : RustTestCaseBase() {
         assertThat(openFiles.find { it.name == "foo.rs" })
             .isNotNull()
     }
+    fun testUnnecessaryParenthesis() {
+        enableInspection<UnnecessaryParenthesisInspection>()
+        myFixture.testHighlighting(true, false, true, fileName)
+
+        for (action in myFixture.getAllQuickFixes()) {
+            if (action.familyName.equals("Remove parenthesis")) {
```

Comment:
thx! i just find that `==` is just the same with `equals` and `is` operator is so powerful in kotlin.